starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,81 @@
-// Names and Input
-
+// Names and input
+var hacker1 = "Bertrand";
+console.log("The driver's name is "+ hacker1);
+var hacker2 = prompt("What is your navigator's name");
+console.log("The navigator's name is "+ hacker2);
 
 //Conditionals
+if (hacker1.length > hacker2.length) {
+  console.log(hacker1+ " has the longest name, it has " + hacker1.length + " characters");
```

Comment:
`//Conditionals
if (hacker1.length > hacker2.length) {
  console.log(
    hacker1 + " has the longest name, it has " + hacker1.length + " characters"
  );
} else if (hacker1.length < hacker2.length) {
  console.log(
    "Yo, " +
      hacker2 +
      " has the longest name, it has " +
      hacker2.length +
      " characters"
  );
} else {
  console.log(
    "wow, you both got equally long names, " + hacker1.length + " characters!!"
  );
}`